app/controllers/course/discussion/posts_controller.rb

```diff
@@ -0,0 +1,106 @@
+class Course::Discussion::PostsController < Course::ComponentController
+  before_action :set_topic
```

Comment:
This method has the same function as this method in 'conditional':
https://github.com/Coursemology/coursemology2/blob/master/app/controllers/course/conditions_controller.rb#L19